examples/deployments/uc_functions/run.py

```diff
@@ -0,0 +1,141 @@
+"""
+# Set environment variables
+export DATABRICKS_HOST="..."
+export DATABRICKS_TOKEN="..."
+export DATABRICKS_WAREHOUSE_ID="..."
+export OPENAI_API_KEY="..."
```

Comment:
What is the key supposed to be set to? Databricks auth token? Can we provide guidance here?